src/canvas/StaticCanvas.ts

```diff
@@ -1693,6 +1694,24 @@ export class StaticCanvas<
       this._objects.length
     } }>`;
   }
+
+  /**
+   * node only method
+   * @throws when called from a browser environment
+   */
+  createPNGStream(opts?: PngConfig) {
+    const impl = getNodeCanvas(this.lowerCanvasEl);
+    return impl && impl.createPNGStream(opts);
+  }
+
+  /**
+   * node only method
+   * @throws when called from a browser environment
+   */
+  createJPEGStream(opts?: JpegConfig) {
+    const impl = getNodeCanvas(this.lowerCanvasEl);
+    return impl && impl.createJPEGStream(opts);
+  }
```

Comment:
look here: https://github.com/Automattic/node-canvas/blob/fdf709a7b08abae33a93c510b96f71df6c13c7b0/types/index.d.ts#L89-L112

We should think of adding `toBuffer` and other methods or might be better to expose a single `getNodeCanvas` method or even better to alter the `getElement` method to return the node canvas instance if running in node. Thoughts?

Reply:
```ts
getElement() {
    getEnv().isLikelyNode
      ? getNodeCanvas(this.lowerCanvasEl)
      : this.lowerCanvasEl;
  }
```

or even as a different method

Reply:
another PR for sure